index.html

```diff
@@ -13,28 +13,154 @@
+      <li><a href="#"><img src="img/m.png"></a></li>
+      <li><a href="#"><img src="img/github.png"></a></li>
+      <li><a href="#"><img src="img/fingers.png"></a></li>
+    </ul>
+
+  </section>
+  <section class="Works">
+    <div class="card">
+      <img src="img/Snapshoot-Portfolio.png" alt="">
+      <h2>Tonic</h2>
+      <ul class="frame2">
+        <li>CANOPY</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>Back End Dev</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="tags">
+        <li class="tag">HTML</li>
+        <li class="tag">CSS</li>
+        <li class="tag">JavaScript</li>
+      </ul>
+      <div class="action">
+        <button>See project</button>
+      </div>
+    </div>
+    <div class="card">
+      <img src="img/Snapshoot-Portfolio1.png" alt="">
+      <h2>Multi-Post Stories</h2>
+      <ul class="frame2">
+        <li>CANOPY</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>Back End Dev</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="tags">
+        <li class="tag">HTML</li>
+        <li class="tag">CSS</li>
+        <li class="tag">JavaScript</li>
+      </ul>
+      <div class="action">
+        <button>See project</button>
+      </div>
+    </div>
+    <div class="card">
+      <img src="img/Snapshoot-Portfolio2.png" alt="">
+      <h2>Tonic</h2>
+      <ul class="frame2">
+        <li>CANOPY</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>Back End Dev</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="tags">
+        <li class="tag">HTML</li>
+        <li class="tag">CSS</li>
+        <li class="tag">JavaScript</li>
+      </ul>
+      <div class="action">
+        <button>See project</button>
+      </div>
+    </div>
+    <div class="card">
+      <img src="img/Snapshoot-Portfolio3.png" alt="">
+      <h2>Multi-Post Stories</h2>
+      <ul class="frame2">
+        <li>CANOPY</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>Back End Dev</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="tags">
+        <li class="tag">HTML</li>
+        <li class="tag">CSS</li>
+        <li class="tag">JavaScript</li>
+      </ul>
+      <div class="action">
+        <button>See project</button>
+      </div>
+    </div>
   </section>
+  <section class="about-myself headline">
+    <h1>About Myself</h1>
```

Comment:
Please change the `h1` and make it an `h2` tag instead. You should have only one h1 tag as the main heading on the page.